platformio.ini

```diff
@@ -13,10 +13,10 @@ platform = espressif8266
 board = esp12e
 framework = arduino
 upload_port = /dev/ttyUSB4
-; upload_port = COM4
 monitor_speed = 115200
-
+monitor_port = /dev/ttyUSB4
```

Comment:
I think you should remove this since it is not portable for other users, you are specifying a port that will not be the same for everyone.